### DISCUSSION TITLE
Not able to install FAST API on Macbook M1

### DISCUSSION THREAD

**Discussion by HimanshuJanbandhu** [fastapi/fastapi#5372] [Questions] [answered]
In the line
`from fastapi import FastAPI, UploadFile, File, HTTPException
`
I am getting the error, ImportError: cannot import name dataclass_transform
<img width="1440" alt="Screenshot 2022-09-11 at 1 58 19 PM" src="https://user-images.githubusercontent.com/42474162/189520208-d4e0bbad-882e-4f62-96e9-dd0cbf310d93.png">
<img width="1440" alt="Screenshot 2022-09-11 at 1 58 24 PM" src="https://user-images.githubusercontent.com/42474162/189520226-2b455675-e678-47ca-bcfc-ecf60869bdd2.png">
I tried to install dataclass_transform seperately but no such package was found, can somebody help

**Comment by iudeen** [comment, marked as answer]
Try to reinstall your FastAPI and its dependencies.

`pip install "fastapi[all]" --upgrade`

As seen in your screenshots, issue is from Pydantic, FYK.

**Comment by tiangolo** [comment]
You might have an old version of Python.

It's probably better to ask in an issue, following the template that will guide you to provide all the information necessary to help you.
